src/github.com/getlantern/idletiming/idletiming_conn.go

```diff
@@ -3,7 +3,10 @@
 package idletiming
```

Comment:
The changes to idletiming are being reviewed [here](https://github.com/getlantern/idletiming/pull/3).